docs/doxygen/other/main_page.dox

```diff
@@ -32,6 +30,7 @@ cmake ..<br />
 make -j4<br />
 sudo make install</code>
 
+\cond
```

Comment:
IIRC this relies on a `@tagname` and an `ENABLED_SECTIONS` tag, but I don't see any of them in this PR. Am I overlooking something?

Reply:
Nevermind, I found this online and it should disable certain sections, but it didn't. I now found out I can simply use HTML comment syntax.